ocs_ci/deployment/helpers/hypershift_base.py

```diff
@@ -62,6 +62,19 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def get_binary_hcp_version():
+    """
+    Get hcp version output. Handles hcp 4.16 and 4.17 cmd differences
+
+    Returns:
+        str: hcp version output
+    """
+    try:
+        return exec_cmd("hcp version").stdout.decode("utf-8").strip()
+    except CommandFailed:
+        return exec_cmd("hcp --version").stdout.decode("utf-8").strip()
```

Comment:
if this try/except is due to 4.16/4.17 version, I suggest to use version comparison and use appropriate version command